frontend/src/modules/fluenteditor/components/rich/RichTranslationForm.js

```diff
@@ -415,31 +422,31 @@ export default function RichTranslationForm(
         let indent = false;
 
         return elements.map((element, eIndex) => {
+            let expression;
             if (
                 element.type === 'Placeable' &&
-                element.expression &&
-                element.expression.type === 'SelectExpression'
+                (expression = element.expression) &&
```

Comment:
Can we just set `const expression = element.expression` above and then just check for `expression` here?

Reply:
Flow already doesn't like that, because `element` can be a `TextElement`, which doesn't have `.expression`. The order of narrowing matters here.